WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,52 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        public static User User { get; set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            Context.Add(User = new User { Id = new Guid("d58893fd-3836-4308-b840-85f4fe548264"), FirstName = "John", LastName = "Doe", Email = "JohnDoe@makinsense.com", UserName = "JohnDoe" });
+            Context.SaveChanges();
+        }
+
+        [TestMethod]
+        public async Task GetByIdAsync_ShouldGetUserById_WhenUserIdExists()
+        {
+            // Arrange
+            IUserService userService = new UserService(Context);

```

Comment:
What if we declare the service as private readonly and initialize it on constructor? Then we can use it directly for each test without arrange it.

Unless we fear that same service instance is not fair to use it on all tests?